metainfo/urllist.go

```diff
@@ -10,18 +10,13 @@ var (
 	_ bencode.Unmarshaler = (*UrlList)(nil)
 )
 
-func (me *UrlList) UnmarshalBencode(b []byte) error {
-	if len(b) == 0 {
-		return nil
+func (me *UrlList) UnmarshalBencode(b []byte) (err error) {
+	if len(b) < 1 {
```

Comment:
I think you've technically changed it to not fail on single byte inputs. This would be much more interesting as a fuzz target or something.